marker/converters/pdf.py

```diff
@@ -53,17 +54,18 @@ def __init__(self, artifact_dict: Dict[str, Any], processor_list: List[str] | No
             processor_list = strings_to_classes(processor_list)
         else:
             processor_list = [
-                FootnoteProcessor,
-                PageHeaderProcessor,
-                EquationProcessor,
-                TableProcessor,
-                SectionHeaderProcessor,
-                TextProcessor,
-                ListProcessor,
+                BlockquoteProcessor,
```

Comment:
just ordered this alphabetically so whatever requires an order just goes at the bottom, like DebugProcessor